src/pages/BrowseSchemes.svelte

```diff
@@ -108,7 +110,7 @@
 
       $map?.fitBounds(bbox(gj), { padding: 20, animate: false });
```

Comment:
We need to blank out the error here, in case the user loads a bad file, then fixes by loading a good one